packages/playground/src/utils/load_deployment.ts

```diff
@@ -220,8 +220,26 @@ export async function loadK8s(grid: GridClient) {
     })
     .filter(item => item && item.masters.length > 0) as K8S[];
   const consumptions = await Promise.all(
-    k8s.map((cluster, index) => {
-      return grids[index].contracts.getConsumption({ id: cluster.masters[0].contractId }).catch(() => undefined);
+    k8s.map(async (cluster, index) => {
+      const contractIds: number[] = [];
+      if (cluster.masters && cluster.masters[0]?.contractId) contractIds.push(cluster.masters[0].contractId);
+      if (Array.isArray(cluster.workers)) {
+        for (const worker of cluster.workers) {
+          if (worker?.contractId) contractIds.push(worker.contractId);
+        }
+      }
+
+      if (contractIds.length === 0) return undefined;
+      try {
+        const reports = await Promise.all(
+          contractIds.map(id => grids[index].contracts.getConsumption({ id }).catch(e => console.log(e))),
```

Comment:
To fail fast, I think we should remove the catch in here.

```suggestion
          contractIds.map(id => grids[index].contracts.getConsumption({ id }))
```